### PR TITLE
Hip Chat notification doesn't include information on what is being alerted on

### PR DESCRIPTION
**Pull request by Samuel1989**
Raised here: https://github.com/grafana/grafana/issues/8426

Provided the option to send plain text messages to Hip Chat rather than just Cards.

Originally the issue was that Cards did not contain information about the series that was in alerting state, I then realised that a Cards description is truncated and could not contain the additional information that would be useful.

To resolve this issue I have added an option to the Alert setup to allow Hip Chat messages to be sent in plain text message format, rather than as cards.

I am still a little uncertain about the terminology, please let me know if you are unhappy with "Plain Text".

New option to turn off cards
screen shot 2017-05-19 at 19 09 33

Original Card Message (which only details alert name)
screen shot 2017-05-19 at 19 10 17

Plain Text (Non Card) Message (which has additional details about what is being alerted on)
screen shot 2017-05-19 at 19 10 34

Truncated Card Message (This was my original attempt, but I backed it out because it was truncated)
screen shot 2017-05-19 at 19 12 25

I have signed the CLA.

Sam

### CODE REVIEW

**Comment by Samuel1989** [comment]
Is anyone able to give some guidance on the failure?

I ran the tests via "grunt test" and "go test" before doing the pull request and they seemed to have passed, however I didn't run "gofmt" which appears to be what is causing the failure?

Any advice/guidance on this would be awesome!

Cheers,
Sam

File: pkg/services/alerting/notifiers/hipchat.go [hunk 1]
```diff
@@ -37,6 +37,15 @@ func init() {
           data-placement="right">
         </input>
       </div>
+			<div class="gf-form max-width-30">
+        <gf-form-switch
+           class="gf-form max-width-30"
+           label="Plain Text"
+           label-class="width-8"
+           checked="ctrl.model.settings.plainText"
+           tooltip="Send Hip Chat messages as text instead of the default card (Allows for more detailed messages).">
```
Comment (torkelo) [inline pkg/services/alerting/notifiers/hipchat.go]: why is does cards not allow more info? are cards only title and image?

Comment (torkelo) [inline pkg/services/alerting/notifiers/hipchat.go]: think we should have one way to send the notification not two

Comment (Samuel1989) [inline pkg/services/alerting/notifiers/hipchat.go]: Hey @torkelo,

I think the reason for both is to allow backwards capability, Hipchat cards are an optional override to the traditional message... Cards are only displayed in supported clients and the message field is used in clients that do not support cards.

Cards allow a title, image and short description, as an observation the short description is truncated and therefore doesn't display the useful information in the alert, however, I haven't found any supporting information that suggests what the limit is.

Sam

Comment (torkelo) [inline pkg/services/alerting/notifiers/hipchat.go]: But if the card override the plain text cannot Grafana send both? So no option is needed?

**Comment by torkelo** [comment]
Do we need the option or can we send both? Seems like horrible solution to not be able to support old clients and new clients at the same time. 

What happens if you send cards & plain text version?

**Comment by bergquist** [comment]
Closing this due to inactivity.
The issue is also solved by https://github.com/grafana/grafana/pull/9111 

thanks for helping out!

**Pull request closed without merging**